config/urls.py

```diff
@@ -5,10 +5,17 @@
 from django.urls import path, re_path
 from django.views import defaults as default_views
 
+from django_otp.admin import OTPAdminSite
 from drf_yasg import openapi
 from drf_yasg.views import get_schema_view
 from rest_framework import permissions
 
+if not settings.DEBUG:
+    admin.site.__class__ = OTPAdminSite
+
+admin.site.__class__ = OTPAdminSite
```

Comment:
We are setting the `OTPAdminSite` regardless of `settings.DEBUG`

Reply:
I did it for testing and leave it there 🤦🏻‍♂️